.github/workflows/post-apiview.yml

```
@@ -0,0 +1,22 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+
+permissions: {}
+
+jobs:
+  event-handler:
+    permissions:
+      statuses: write
+      pull-requests: write
+      checks: read
+    name: Handle ${{ github.event_name }} ${{ github.event.action }} event
+    runs-on: ubuntu-latest
+    steps:
```

```suggestion
    if: github.event.check_suite.app.name == 'Your Desired Suite Name' # Replace with actual suite name
    steps:
```

You probably want to limit this to one particular check_suite, not all suites.  For "vanilla" pipelines I think the suite name is "Azure Pipelines".